woocommerce/woocommerce
# Remove Cart and Checkout templates migration code

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/44124.

This PR removes code used to migrate the Cart and Checkout pages into the templates. As mentioned in #44124, this is no longer needed as those templates use the `page-content-wrapper` block to render the content of the pages. Tests for the `page-content-wrapper` block were added in https://github.com/woocommerce/woocommerce/pull/44122.

### How to test the changes in this Pull Request:

1. Make an edit to the Cart or Checkout pages.
2. With a block theme, go to Appearance > Editor > Templates and open the Page: Cart and Page: Checkout templates.
3. Verify the edits you did in step 1 are visible.
4. Now, visit the cart and checkout in the frontend.
5. Verify again the edits you did in step 1 are visible.

### Changelog entry


<details>

#### Significance


#### Type




#### Message 

#### Comment

Remove Cart and Checkouts templates migration code

</details>
